src/ComponentInstantiationTransformer.cpp

```diff
@@ -508,37 +508,37 @@ bool ComponentInstantiationTransformer::transform(AstTranslationUnit& translatio
     // remember the remaining orphan clauses
     program.clauses.clear();
     program.clauses.swap(unbound);
+    /*
```

Comment:
Why is this here but commented out? Can this be removed? Or is there an issue with the current implementation that needs this as a reference?

Reply:
It should be removed. AstRelations doesn't store related AstStore or AstLoad any more.

Reply:
OK, I will remove it in my next PR.